### ISSUE TITLE
Is it possible to use a different editor

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#6536]
We want to extend the way we create pages and allow for more dynamic inner pages with inner menus. Like being able to create small inner websites. I am trying to use wordpress as underlying CMS to create more complex pages, but working from subdirectory url is some times hard because I don't want to display the whole WP site, just certain pages. As those pages need to run inside an iframe, this some times causes issues when we try to use code within the footer area. We can get it to work, just not as seemless as we want it too.

Pages created within AVideo editor as articles are much easier to work with, but are somewhat a bit restricted in the way we want those pages to look like. 

Is there a way we can use a different editor like TinyMCE?
https://cdn.tiny.cloud/1/b0uhobddgiyffk5krgppa24maon93zhf61vnuvpqq48026z0/tinymce/6/tinymce.min.js

OR

Is there a way for me to open my iframe links which I set in the Footer HTML code area, and have those links open under the AVideo iframe so those new opened pages will appear under the AVideo top menu bar.

If I user target="_parent" or "_top" those pages will open without that AVideo iframe and I don't want to use _blank.
If can solve that, is would be awesome to integrate some sort of templating system.
So we would like to see AVideo also being able to have more flexibility with those articles if possible and create more complex content and articles.

**Comment by guymass** [comment]
Another solution would be to copy the top menu to the header of the CMS.

**Comment by DanielnetoDotCom** [comment]
We can use TinyMCE, but, is it to replace what editor exactly? can you please screenshot it?

**Comment by guymass** [comment]
I have found a solution for it, we are now integrating Wordpress as an underlying content creation for extra content like an eshop for example. Now we can create pages and run them under the site's iframe. The way to do it is to create menus in the TopMenu and set them to iframe and non-active. Once the menu is created, it generates a new URL and this can be used to view those pages from a simple HTML menu we set in the "Under the top menu" area under the "customize advance" plugin.

So in short we don't need to change the editor anymore.
